ExampleCode/C++/Apps/README.md

```diff
@@ -0,0 +1,9 @@
+C++ Example Apps
+================
+
+These are example apps using the C++ iSENSE API wrapper.
+
+The following apps currently exist:
```

Comment:
Keep:
`These are example apps using the C++ iSENSE API wrapper.`
Remove:
`The following apps currently exist:`